### PR TITLE
fixed the layout

### PR DESCRIPTION
**Pull request by MaiCodes-exe**

### CODE REVIEW

**Review by ertip4geek — APPROVED** [review]

**Comment by ertip4geek** [comment]
Maise sorry last thing, for your pending PR, could you make 1 more commit without the file package-lock.json.
Because is causing 12 conflicts and is doing errors to be resolvev, just skip that file will be ok
thanks!:

File: src/front/js/pages/contacts.js [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      <div className="text-center" id="titulo">
+        <h3> Our Mission</h3>
+      </div>
+      <Container className="container-fluid">
+        <Row>
+          <Col classname="col-5">
+            <img
+              src="https://images.pexels.com/photos/5691814/pexels-photo-5691814.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=650&w=940"
+              id="kids"
+            ></img>
```
Comment (vitortamberlini) [inline src/front/js/pages/contacts.js:20]: is a self-closing tag:
```suggestion
            <img
              src="https://images.pexels.com/photos/5691814/pexels-photo-5691814.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=650&w=940"
              id="kids"
            />
```

File: src/front/js/pages/contacts.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+          </Col>
+          <Col>
+            <p className="row-md col-6" id="mission">
+              NannyHub was born from the vision of one of our members, as a
+              parent. At the begging, as a group, we decided to follow this
+              vision, considering 2 out of 3 members are parents and feel very
+              strongly motivated to improve the nanny and childcare industry.
+              Later on, we were joined by another memember who, despite not
```
Comment (vitortamberlini) [inline src/front/js/pages/contacts.js:28]: Typo:
```suggestion
              Later on, we were joined by another member who, despite not
```

File: src/front/styles/contacts.css [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+  margin-left: 15%;
+}
+
+#contact {
+  margin-left: 10%;
+  letter-spacing: 1px;
+  line-height: 1.8;
+}
+
+#titulo {
```
Comment (vitortamberlini) [inline src/front/styles/contacts.css:18]: nit: Change this id to `title` for consistency

**Review by vitortamberlini — CHANGES_REQUESTED** [review]

**Pull request merged**